src/include/connection.h

```diff
@@ -329,12 +329,14 @@ struct __wt_connection_impl {
     wt_off_t ckpt_logsize; /* Checkpoint log size period */
     bool ckpt_signalled;   /* Checkpoint signalled */
 
-    uint64_t ckpt_apply;      /* Checkpoint handles applied */
-    uint64_t ckpt_apply_time; /* Checkpoint applied handles gather time */
-    uint64_t ckpt_skip;       /* Checkpoint handles skipped */
-    uint64_t ckpt_skip_time;  /* Checkpoint skipped handles gather time */
-    uint64_t ckpt_usecs;      /* Checkpoint timer */
-    uint64_t ckpt_prep_max;   /* Checkpoint prepare time min/max */
+    uint64_t ckpt_apply;            /* Checkpoint handles applied */
+    uint64_t ckpt_apply_time;       /* Checkpoint applied handles gather time */
+    uint64_t ckpt_apply_saved_list; /* Checkpoint handles that used saved ckptlist */
+    uint64_t ckpt_apply_meta_list;  /* Checkpoint handles that built ckptlist from metadata */
```

Comment:
Alphabetize the fields here.

Reply:
@sulabhM this is not yet completed.

Reply:
Sorry about that, failed to save the file after changing it. It should be done now.